#!/usr/bin/env python
PACKAGE = "masterslave"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("rotGain", double_t, 0, "rotational gain", 0.002, 0, 1)
gen.add("transGain", double_t, 0, "translational gain", 0.001, 0, 1)
gen.add("joyThresh", double_t, 0, "joystick threshold", 0.02, 0, 1)
exit(gen.generate(PACKAGE, "controldevice", "controldevice"))
